### DISCUSSION TITLE
ipv6无效

### DISCUSSION THREAD

**Discussion by wolvth** [istoreos/istoreos#646] [Q&A] [answered]
2023.9.3
咋又被顶上来了。
**因为新固件我一直没尝试，之前的ipv6是没问题的，当时可能是影响dns类的插件折腾过多所引起**
在光猫运营商方面没问题的情况下，出问题的建议大家重刷，同时终端禁用并启用一次再尝试。
推荐几个检测方法：
1，找支持ipv6测速的网站，比如
https://test.nju.edu.cn/
http://test6.nju.edu.cn/
能检测ipv6的dns解析，下载访问正常如否
2，找ipv6的电视直播源
我这里随便给一条，可以检测ipv6直连的效果，**关闭与开启ipv6的DNS解析不影响正常播放ipv6直播源**
#EXTINF:-1 ,CCTV-13
http://[2409:8087:6a01:1020::c]/cdnrrs.gz.chinamobile.com/PLTV/88888888/224/3221225748/1/index.m3u8?fmt=ts2hls

**Comment by jjm2473** [comment, marked as answer]
ipv6地址应该没问题，但是默认关闭了DNS的ipv6解析，以免网页加载慢，如果要开启ipv6解析，可以到“DHCP/DNS”的高级设置配置

**Reply by zx09** [reply]
目前最新固件 设置了也不行

**Reply by jjm2473** [reply]
是没有ipv6地址，还是ipv6网站连不上？

**Reply by zx09** [reply]
pc网卡能看到240开头的移动ipv6地址 但是无法通过ipv6测试 手机4g流量无法访问路由器wan的ipv6 但是pc可以访问ddns域名 我之前用过esir的openwrt正常配置过ipv6与ddns esir固件都能正常使用ipv6 我下了[istoreos]0324的版本 后面在线升级了0407版 ipv6都不行

**Reply by jjm2473** [reply]
@zx09 开启DNS的ipv6解析后也不行吗？

**Reply by zx09** [reply]
对的 我用过esir的固件 知道要开ipv6解析

**Comment by fujie-xiyou** [comment]
我这边测试，打开dns高级设置中的ipv6解析后，并且在防火墙中放开ipv6 icmp协议之后，test-ipv6可以全过
![image](https://user-images.githubusercontent.com/23140941/233657393-3e2e8d88-8de1-4817-83ac-b40d8848dd82.png)

**Reply by jjm2473** [reply]
icmpv6本身就允许啊，允许每秒钟1000个包

**Comment by Jie-baobao** [comment]
遇见同样问题了，最新版的系统ipv6还是不能用，有地址访问不了，但是路由器wan口分配的倒是能用。内部电脑分配的就不行了，访问不了

**Reply by jjm2473** [reply]
试试拨号

**Reply by jjm2473** [reply]
如果你的wan口都没ipv6-pd，可能lan口就没法再往下分配了

**Reply by wolvth** [reply]
咋又被顶上来了。
因为新固件我一直没尝试，之前的ipv6是没问题的，当时可能是影响dns类的插件折腾过多所引起的。
在光猫运营商方面没问题的情况下，出问题的建议大家重刷，同时终端禁用并启用一次再尝试。
推荐几个检测方法：
1，找支持ipv6测速的网站，比如
https://test.nju.edu.cn/
http://test6.nju.edu.cn/
能检测ipv6的dns解析，下载访问正常如否
2，找ipv6的电视直播源
我这里随便给一条，可以检测ipv6直连的效果，关闭与开启ipv6的DNS解析不影响正常播放ipv6直播源
#EXTINF:-1 ,CCTV-13
http://[2409:8087:6a01:1020::c]/cdnrrs.gz.chinamobile.com/PLTV/88888888/224/3221225748/1/index.m3u8?fmt=ts2hls

**Reply by Jie-baobao** [reply]
wan口的ipv6正常用，拨号也试了都没问题，就是往lan口分的就不行

**Reply by jjm2473** [reply]
@Jie-baobao 看有没有分配到ipv6-pd地址

**Reply by SunLee7** [reply]
我也遇到了 有ipv6-pd

**Reply by Jie-baobao** [reply]
我已经解决了。换了eSir的固件。我解决不了ipv6只能换个系统了。

**Comment by SunLee7** [comment]
我发现在istore安装多线多播之后lan口ipv6就不好使了，卸载了就好了🧐

**Reply by jjm2473** [reply]
主要是mwan3插件会导致ipv6的问题
